src/components/core/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState, ReactNode, HtmlHTMLAttributes } from 'react';
+import * as CheckboxPrimitive from '@radix-ui/react-checkbox';
+import classNames from 'classnames';
+import css from './checkbox.module.scss';
+import CheckIconSVG from '../../../images/svg/checkbox-icon.svg';
+import CheckIndetermineIconSVG from '../../../images/svg/checkbox-indetermine-icon.svg';
+
+export const CheckBox = CheckboxPrimitive.Root;
+
+export interface CheckboxProps extends HtmlHTMLAttributes<HTMLInputElement> {
+    dark?: boolean;
+    indetermine?: boolean;
+    check?: boolean;
+    size?: 'default' | 'small';
+}
+
+const Checkbox = ({ children, dark, indetermine, check, size = 'default', ...props }: CheckboxProps) => {
+    return (
+        <div className={classNames(css.container)} {...props}>
+            <CheckBox
+                defaultChecked={check}
+                className={classNames(css.checkbox, dark && css.dark, check && css.check, indetermine && css.check)}
+            >
+                <>
+                    {check && <img src={CheckIconSVG} alt="" />}
+                    {indetermine && <img src={CheckIndetermineIconSVG} alt="" />}
```

Comment:
```suggestion
                    {check && <img src={CheckIconSVG} alt="" />}
                    {indetermine && <img src={CheckIndetermineIconSVG} alt="" />}
```
Do we need to add an `alt` here?

Reply:
Resolved in latest commit